packages/dioxus-tui/examples/components.rs

```diff
@@ -1,67 +0,0 @@
-#![allow(non_snake_case)]
```

Comment:
This example was just a better version of the quadrants example, so this PR moves it into the quadrants example